packages/main/test/pages/Kitchen.html

```diff
@@ -10,6 +10,12 @@
 
 	<link rel="stylesheet" type="text/css" href="./styles/Kitchen.css">
 
+	<style>
+		ui5-avatar img {
```

Comment:
The follow the project conventions is better to style the components using the atttribute instead of the tag:

```css
[ui5-avatar] img {
	object-fit: contain;
}
```